calebwagner/react-notes-app
# Cjw styling

#### Changes Made
1. Modified file `all files` to include comments
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-styling
    ```
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
